libs/brig-types/src/Brig/Types/Swagger.hs

```diff
@@ -440,8 +446,12 @@ teamInvitation = defineModel "TeamInvitation" $ do
   property "created_by" bytes' $ do
     description "ID of the inviting user"
     optional
-  property "name" string' $
+  property "invitee_name" string' $ do
     description "Name of the invitee"
+    optional
+  property "phone" string' $ do
+    description "E.164 Phone number of the invitee"
```

Comment:
What does E.164 mean?

Reply:
That's just the international format, including the `+` sign.